CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ enable_language(CXX)
 set (CMAKE_CXX_STANDARD 11)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
+find_package(SDL2 REQUIRED)
```

Comment:
@diamant3 this cannot go there, this is the cmake of the library which does not depend on SDL2.
look inside the examples folder, there is a cmake file for the examples project, where it should reference sdl2

Reply:
alright, i will remove that